lib/solidus_backtracs/api/request_runner.rb

```
@@ -12,14 +12,39 @@ def initialize
         @api_base = ENV['BACTRACS_API_BASE']
```

should throw a noise log if env vars missing, unless TEST/DEV env, since this will break the integration